Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs

```diff
@@ -40,6 +40,7 @@ public SiteExtensionInfo(SiteExtensionInfo info)
             LocalIsLatestVersion = info.LocalIsLatestVersion;
             LocalPath = info.LocalPath;
             InstalledDateTime = info.InstalledDateTime;
+            InstallationArgs = info.InstallationArgs;
```

Comment:
It will depend. Kudu ignores the installation request if the extension version is not changed so the user must uninstall the extension in order to provide the updated arg set.
If the version is changed, then Kudu will apply the new extension installation as an update mechanism (which will call the install.cmd with the arg set).